app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :confirmable, :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] One of the major requirements for this project is for a user to be able to confirm the email they used for registration. Currently, your application doesn't support this feature because I noticed that once I sign up, I can have access to the `Users#Index` page without confirming the email. Kindly activate the `:confirmable` devise to enable this feature. 👍 😎